assemblyline/odm/models/submission.py

```diff
@@ -44,46 +49,47 @@ class ServiceSelection(odm.Model):
+    malicious = odm.Boolean(default=False, description="**TODO**: **Original**:Is the file submitted already known to be malicious? **Generated**:Flags the submission as known to be malicious, possibly altering its handling.")
+    max_extracted = odm.Integer(default=500, description="The maximum number of files that can be extracted from the submission for separate analysis.")
+    max_supplementary = odm.Integer(default=500, description="**TODO**: **Original**:Max number of supplementary files **Generated**:The maximum number of supplementary files that can be generated from the submission.")
+    priority = odm.Integer(default=1000, description="**TODO**: **Original**:Priority of the scan  **Generated**:The processing priority of the submission, with lower numbers indicating higher priority.")
+    profile = odm.Boolean(default=False, description="**TODO**: **Original**:Should the submission do extra profiling?  **Generated**:Triggers additional profiling of the submission for performance and analysis metrics.")
+    psid = odm.Optional(odm.UUID(), description="The ID of a parent submission, if this submission is related to or derived from another.")
+    quota_item = odm.Boolean(default=False, description="Indicates if this submission should count against the submitter's quota.")
+    services = odm.Compound(ServiceSelection, default={}, description="Details which services are selected or excluded from processing this submission.")
     service_spec = odm.Mapping(odm.Mapping(odm.Any()), default={}, index=False, store=False,
-                               description="Service-specific parameters")
-    submitter = odm.Keyword(store=True, copyto="__text__", description="User who submitted the file")
-    ttl = odm.Integer(default=0, description="Time, in days, to live for this submission")
-    type = odm.Keyword(default="USER", description="Type of submission")
-    initial_data = odm.Optional(odm.Text(index=False), description="Initialization for temporary submission data")
+                               description="A dictionary specifying service-specific parameters that may alter service behavior for this submission.")
+    submitter = odm.Keyword(store=True, copyto="__text__", description="The username of the individual who submitted the file for analysis.")
+    ttl = odm.Integer(default=0, description="**TODO**: **Original**:Time, in days, to live for this submission  **Generated**:The time-to-live for the submission, defining how many days it will be retained before expiry.")
+    type = odm.Keyword(default="USER", description="The type of submission (e.g., `USER` for user-submitted), indicating its origin or purpose.")
+    initial_data = odm.Optional(odm.Text(index=False), description="Initial temporary data provided at the time of submission, which may be used during analysis.")
```

Comment:
```suggestion
    initial_data = odm.Optional(odm.Text(index=False), description="Content provided at the time of submission to initialize the temporary submission data, which may be used during analysis.")
```
The word group "Temporary submission data" is important, and should be used a a reference for everything that interacts with the temporary submission data.